src/H5FDpublic.h

```diff
@@ -178,6 +179,12 @@
  * operations when appropriate.
  */
 #define H5FD_FEAT_MEMMANAGE 0x00010000
+/*
```

Comment:
This should be 'H5FD_FEAT_SUPPORTS_SWMR', and have a better comment about what that means.